### DISCUSSION TITLE
Commits to my public repository are not showing up as contributions. I feel like I've tried everything, what am I missing?

### DISCUSSION THREAD

**Discussion by sgiori11** [community/community#48668] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm building a NextJS site using create-next-app, and none of my commits are showing up in my contribution graph. I have checked all of the following: 

- My commit email matches the email in my github profile (I checked this with commit url.patch)
- My commit username matches my github username 
- I'm working in [my own, public repository](https://github.com/sgiori11/it-startups-job-board) 
- All of my commits are on the Main branch 

What am I missing? Why are none of my commits from the last 2 weeks showing up on my github profile? :(

**Comment by airtower-luna** [comment, marked as answer]
None of the commits in the repository are recognized as yours. It looks like the problem is that the author email in the commit includes quotes around the actual mail address. Remove those and it should work, assuming the email address is otherwise correct.

**Reply by sgiori11** [reply]
THANK YOU!!

I just assumed my terminal was putting quotes around the email because it was a response...omg
